cmd/satellite/main.go

```diff
@@ -9,10 +9,10 @@ import (
 	"path/filepath"
 
 	"github.com/spf13/cobra"
-	"storj.io/storj/internal/mock"
 	"storj.io/storj/pkg/cfgstruct"
 	"storj.io/storj/pkg/kademlia"
 	"storj.io/storj/pkg/overlay"
+	mock "storj.io/storj/pkg/overlay/mocks"
```

Comment:
call the package mockoverlay, then you can import it as is

Reply:
alternatively name it mock and use as is

Reply:
https://blog.golang.org/package-names, but yeah I think mockoverlay will avoid some potential conflicts in future.